### ISSUE TITLE
Does the plugin support no studio install

### ISSUE THREAD

**Issue by mshahat** [TIBCOSoftware/bw6-plugin-maven#129]
Hi, 

Does the plugin support no studio install ? in case of a CI server agent i would like to know whether we can run the plugin without needing a studio installation ? If yes, can we have the instructions ? 

Thanks,

**Comment by ashutoshbhide** [comment]
Hi @mshahat 

Yes, once the POM files are generated, there is no dependency on Studio of TibcoHome to generate the EAR files. 
You can run the maven commands directly on the parent project (without installing BW) as long as the system has Maven installed.
There is no specific instruction for it. Its standard maven way. You can configure the same in Jenkins or any CI server and run it.

The Maven Plugin is designed in such a way to avoid any TibcoHome dependency. 

Ashutosh

**Comment by mshahat** [comment]
Thanks @ashutoshbhide. let me rephrase my question: 
what is the plugin installation procedure on a CI server agent ? that wouldn't have a TIBCO home ? 

Thanks

**Comment by mshahat** [comment]
Hi, any update to my question please. i'd like to build a build server where i don't have a Studio install , how can i get the maven plugin installed ? 
@ashutoshbhide 
Thanks,

**Comment by awagle** [comment]
Unzip **bw6-maven-plugin.zip** from under the **config** folder in the installation. It has two install scripts using which you can use to push the maven plugin to a repository.
